src/index.ts

```diff
@@ -93,12 +93,22 @@ async function createKubernetesDeployment(
     const release = inputs.release || getReleaseFromImage(inputs.image);
     const namespace = inputs.namespace || "default";
     const host = resolveHost(inputs.host, app, release);
-    const env = ((inputs.env || "")
```

Comment:
WTF? 😂

Reply:
The git diff is acting strangely here xD. The statement continued at line 99 with `.map(key => /* ... */` 😅